articles/design/creating-invite-only-applications.md

```diff
@@ -1,118 +1,137 @@
+            <p>If it wasn't you please disregard this email and make sure you can still login to your account. If it was you, then to <strong>confirm the password change <a href="{{ url }}">click here</a></strong>.</p>
 
-```har
-{
-	"method": "POST",
-	"url": "https://${account.namespace}/api/v2/tickets/email-verification",
-	"httpVersion": "HTTP/1.1",
-	"cookies": [],
-	"headers": [{
-		"name": "Authorization",
-		"value": "Bearer MGMT_API_ACCESS_TOKEN"
-	}],
-	"queryString": [],
-	"postData": {
-		"mimeType": "application/json",
-		"text": "{ \"user_id\": \"abcd|1234\", \"client_id\": \"xyz789\" }"
-	},
-	"headersSize": -1,
-	"bodySize": -1,
-	"comment": ""
-}
+    {% endif %}
+
+            <p>If you have any issues with your account, please don’t hesitate to contact us at 1-888-AWESOMECO.</p>
+
+            <br>
+            Thanks!
+            <br>
 ```
 
-### Password Reset
-
-Once you've verified the user's email, you will need to initiate the [password change process](/connections/database/password-change). To do so, your app should make a `POST` request to Auth0's Management API.
-
-Be sure to replace the placeholder values for your [API Access Token](/api/management/v2/tokens), as well as those within the body of the call, including the callback/return URL for your app and the user's details.
-
-```har
-{
-	"method": "POST",
-	"url": "https://${account.namespace}/api/v2/tickets/password-change",
-	"httpVersion": "HTTP/1.1",
-	"cookies": [],
-	"headers": [{
-		"name": "Authorization",
-		"value": "Bearer MGMT_API_ACCESS_TOKEN"
-	}],
-	"queryString": [],
-	"postData": {
-		"mimeType": "application/json",
-		"text": "{ \"result_url\": \"YOUR_APP_CALLBACK_URL\", \"user_id\": \"USER_ID\", \"new_password\": \"secret\", \"connection_id\": \"con_0000000000000001\", \"email\": \"EMAIL\", \"ttl_sec\": 0 }"
-	},
-	"headersSize": -1,
-	"bodySize": -1,
-	"comment": ""
+```js
+function (user, context, callback) {
+  const request = require('request');
+  const userApiUrl = auth0.baseUrl + '/users/';
+
+  // This rule is only for Auth0 databases
+  if (context.connectionStrategy !== 'auth0') {
+    return callback(null, user, context);
+  }
+
+  if (user.email_verified || !user.last_password_reset) {
+    return callback(null, user, context);
+  }
+
+  // Set email verified if a user has already updated his/her password
+  request.patch({
+    url: userApiUrl + user.user_id,
+    headers: {
+      Authorization: 'Bearer ' + auth0.accessToken
+    },
+    json: { email_verified: true },
+    timeout: 5000
+  },
+  function(err, response, body) {
+    // Setting email verified isn't propagated to id_token in this
+    // authentication cycle so explicitly set it to true given no errors.
+    context.idToken.email_verified = (!err && response.statusCode === 200);
+
+    // Return with success at this point.
+    return callback(null, user, context);
+  });
```

Comment:
The rule to set the email as verified is no longer needed.  The Change password ticket logic can do this now and is covered above